packages/components/src/components/counter/counter.tsx

```diff
@@ -1,14 +1,12 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 
-const Counter = ({ className, count }) => {
-    return <div className={classNames('dc-counter', className)}>{count}</div>;
+type TCounterProps = {
+    className: string;
+    count: number;
 };
```

Comment:
@iman-fs, you can extend the native `HTMLDivElement`, so that you don't have to redeclare `className`. This will also future proof any native props that will be added. It'll also provide IntelliSense whilst using the component

Reply:
thanks @jim-deriv . updated my code